css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+form {
+  width: 50%;
+  margin: auto;
+}
+
+h3 {
+  text-align: center;
+  color: #ee6c66;
+  text-transform: uppercase;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: 900;
+  font-weight: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
+}
+
+label {
+  padding-left: 0.5em;
+  text-transform: uppercase;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
+  font-size: 65%;
+}
+
+div {
+  padding-bottom: 0.5em;
+}
+
+.user-input {
+  width: 100%;
+  padding: 0.5em;
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
+  border: 1px solid lightgray;
+  border-radius: 5px;
+}
+
+.submit-div {
+  /* display: inline-block; */
+  text-align: right;
+  width: 100%;
+}
+
+.submit-button {
+  font-size: 12px;
+  padding: 10px 20px;
+  background-color: #ec5d57;
+  color: white;
+  border-radius: 20px;
+  border: none;
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: bolder;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
+}
+
+.engineer-input {
+  width: 100%;
+  padding: 0.5em;
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
```

Comment:
These are only necessary for the `@font-face` rules, you don't need to worry about them. You can just use the font with the `font-family` property.